docs/source/faq.rst

```diff
@@ -3,18 +3,36 @@ FAQ
 
 .. py:currentmodule:: htmap
 
-.. _install:
+.. _successful-jobs:
 
-How do I install HTMap?
------------------------
+How do I collect only completed jobs?
+-------------------------------------
 
-On Unix/Linux systems, running ``pip install htmap`` from the command line should suffice.
-On Windows, you may need to install HTCondor itself to get access to the HTCondor Python bindings, and use the ``no-deps`` option with ``pip install``.
-You only need to do this "submit-side", but you may also need to do some work to make sure your code will run execute-side: see :doc:`dependencies`.
+Let's say you submitted 10,000 long-running jobs, and 99.9% of these jobs
+complete successfully. You'd like to get the results from the successful jobs,
+and save the results to disk.
 
-* To get the latest development version of HTMap, run ``pip install git+https://github.com/htcondor/htmap.git`` instead.
-* Run ``pip install git+https://github.com/htcondor/htmap.git@<branch>`` to install a specific branch.
-* You may need to append ``--user`` to the ``pip`` command if you do not have permission to install packages directly into the Python you are using.
+The right function to use is :func:`~htmap.Map.components_by_status`. It can
+filter out the successful jobs and process those. See the
+:func:`~htmap.Map.components_by_status` documentation for an example usage.
+
+
+Is it possible to use Dask with HTCondor?
+-----------------------------------------
+
+`Dask Distributed`_ is a lightweight library for distributed Python computation.
+Dask Distributed has familiar APIs, is declarative and supports more complex
+scheduling than map/filter/reduce.
+
+`Dask-Jobqueue`_ present a wrapper for HTCondor clusters through their
+`HTCondorCluster`_. After `HTCondorCluster`_ is used, Dask can be used as
+normal or on your own machine. This is common with other cluster managers too:
+Dask-Jobqueue also wraps SLURM, SGE, PBS and LSF clusters, and Dask Distributed
+can wrap Kubernetes and Hadoop clusters. This interface will not use HTMap.
```

Comment:
There are no downsides mentioned in use of `HTCondorCluster`. Should some be added?

Reply:
Thanks. I'll let you do that – you'll write a better reason than me.